resources/templates/messages.html

```diff
@@ -0,0 +1,24 @@
+<html>
+<body>
+
+<form action="/create-message" method="post">
+    <input type="text" maxlength="140" placeholder="Enter your text" name="messageText"/>
+    <div id="content"> {{{content}}} </div>
+    <button type ="send">Send</button>
+
+
```

Comment:
Did you figure out how to display the messages?  Please add that code and resubmit.